packages/next/server/config-shared.ts

```diff
@@ -145,6 +145,12 @@ export interface ExperimentalConfig {
   }
   swcPlugins?: Array<[string, Record<string, unknown>]>
   largePageDataBytes?: number
+  /**
+   * If set to `false`, webpack won't fall back to polyfill Node.js modules in the browser
+   * Full list of old polyfills is accessible here:
+   * [webpack/webpack#ModuleNotoundError.js#L13-L42](https://github.com/webpack/webpack/blob/2a0536cf510768111a3a6dceeb14cb79b9f59273/lib/ModuleNotFoundError.js#L13-L42)
+   */
+  fallbackNodePolyfills?: boolean
```

Comment:
should it also set to `true` by in `defaultConfig`? falsy values like `undefined` and `false` should behave same and `true` will always use the polyfills

Reply:
Should not be necessary since we match against `false`: https://github.com/vercel/next.js/pull/39248/files#diff-35e60cb9cfc6f14c2998799e7e5a28e5adfd6ededc273851531da3a1e506b0d4R1581

Reply:
I updated the type to only allow setting `false`